### PR TITLE
Updates for Uno 8.7

### PR DESCRIPTION
**Pull request by sofietoft**

### CODE REVIEW

File: Umbraco-Uno/Widgets/Text/index.md [hunk 1]
```diff
@@ -4,22 +4,29 @@ versionFrom: 8.0.0
...
 - Sidebar widgets
 
-![Text Backoffice](images/Text-Backoffice1.png)
+### Settings
+
+- Container Width
+- Sidebar Left
 
 ::: Note
 The Text widget is the *standard* width per default. This cannot be changed.
```
Comment (sofietoft) [inline Umbraco-Uno/Widgets/Text/index.md:32]: You say earlier in this article that "you will have the option to choose the size of your text box".
Doesn't that contradict what it says in this box? 🤔

Comment (sofietoft) [inline Umbraco-Uno/Widgets/Text/index.md:32]: @ctf-umbraco 
It's said earlier in this article that "you will have the option to choose the size of your text box".
Doesn't that contradict what it says in this box? 🤔

**Review by sofietoft — COMMENTED** [review]
Checking viewed files.

File: Umbraco-Uno/Widgets/Text/index.md (hunk 1 above)
Comment (ctf-umbraco) [reply Umbraco-Uno/Widgets/Text/index.md:32]: yes it does this is a typo it should say that it can be changed within the settings

**Pull request merged**